libuavcan_drivers/stm32/driver/include/uavcan_stm32/thread.hpp

```diff
@@ -161,6 +167,40 @@ class Mutex
     void unlock() { }
 };
 
+#elif UAVCAN_STM32_FREERTOS
+
+class BusEvent
+{
+    SemaphoreHandle_t sem_;
+    BaseType_t xHigherPriorityTaskWoken;
```

Comment:
I think that should stay class variable because i'm using it in function yieldFromISR. @pavel-kirienko